Wikipedia/Code/ExploreViewController.swift

```diff
@@ -24,6 +24,10 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
         NotificationCenter.default.addObserver(self, selector: #selector(exploreFeedPreferencesDidSave(_:)), name: NSNotification.Name.WMFExploreFeedPreferencesDidSave, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(articleDidChange(_:)), name: NSNotification.Name.WMFArticleUpdated, object: nil)
         NotificationCenter.default.addObserver(self, selector: #selector(articleDeleted(_:)), name: NSNotification.Name.WMFArticleDeleted, object: nil)
+        
+        if UserDefaults.standard.wmf_isFastlaneSnapshotInProgress() {
```

Comment:
This and any other fastlane checks should be wrapped in `#if TEST` so they compile out on release builds

Reply:
Fixed!